indi-qhy/qhy_focuser.h

```diff
@@ -29,47 +33,57 @@ class QFocuser : public INDI::Focuser
     virtual void TimerHit() override;
 
 private:
-  // Connection::Serial *serialConnection{nullptr};
   int SendCommand(char *cmd_line);
   int ReadResponse(char *buf);
   void GetFocusParams();
 
   int updatePosition(double *value);
   int updateTemperature(double *value);
-  int updateBacklash(double *value);
-  int updateMotorSettings(double *duty, double *delay, double *ticks);
   int updatePositionRelativeInward(double value);
   int updatePositionRelativeOutward(double value);
   int updatePositionAbsolute(double value);
 
-  int updateMaxPosition(double *value);
   int updateSetPosition(int value);
 
-  int updateSetSpeed(int value);
+  int updateSetReverse(int value);
 
-  int ReadUntilComplete(char *buf, int timeout);
+  int updateSetSpeed(int value);
 
   int timerID { -1 };
   bool initTargetPos = true;
   double targetPos{ 0 };
   double simulatedTemperature{ 0 };
   double simulatedPosition{ 0 };
+  bool isReboot = false;
+  int RebootTimes = 0;
 
   uint8_t buff[USB_CDC_RX_LEN];
 
+  int32_t cmd_version;
+  int32_t cmd_version_board;
   int32_t cmd_position;
   int32_t cmd_out_temp;
   int32_t cmd_chip_temp;
+  int32_t cmd_voltage;
+
 
   ISwitch ResetPosS[1];
   ISwitchVectorProperty ResetPosSP;
```

Comment:
1. Please use INDI::PropertySwitch
2. How is this different from focuser SYNC? Sync takes whatever value and resets the focuser to this position.